pages/common/leave.md

```diff
@@ -2,6 +2,7 @@
 
 > Set a reminder for when it's time to leave.
 > To remove reminders use `kill $(pidof leave)`.
+> More information: <https://www.freebsd.org/cgi/man.cgi?query=leave>.
```

Comment:
Why not use <https://manned.org/leave> here?

Reply:
Because it's the only official source I found, and its available options does not change at all.

Reply:
FreeBSD is its official source, but it is available on Linux and macOS (and therefore in `common`)? What?

Reply:
I couldn't find anyone maintaining this package in Linux. This command originates from BSD and freeBSD's website includes it, so I think freeBSD's man page is a better source than manned.org.

Reply:
And `manned.org` is getting the man page from the BSD manual.